core/metacling/src/TCling.cxx

```diff
@@ -1160,6 +1164,10 @@ TCling::TCling(const char *name, const char *title)
    }
 #endif
 
+#ifdef R__FAST_MATH
+      interpArgs.push_back("-ffast-math");
```

Comment:
`interpArgs` already contains `clingArgsStorage`, see line 1150. Does everything work if you remove this hunk?

Reply:
it work, if remove  clingArgsStorage
 ```
 #ifdef R__FAST_MATH
      clingArgsStorage.push_back("-ffast-math");
#endif 
```
but not other...
I recomment clean code!